vizro-core/tests/unit/vizro/models/_components/form/test_checklist.py

```diff
@@ -148,9 +128,18 @@ def test_set_action_via_validator(self, identity_action_function):
 class TestChecklistBuild:
     """Tests model build method."""
 
-    def test_checklist_build(self, expected_checklist):
-        checklist = Checklist(options=["A", "B", "C"], id="checklist_id", title="Title").build()
-
-        result = json.loads(json.dumps(checklist, cls=plotly.utils.PlotlyJSONEncoder))
-        expected = json.loads(json.dumps(expected_checklist, cls=plotly.utils.PlotlyJSONEncoder))
-        assert result == expected
+    def test_checklist_build(self):
+        checklist = Checklist(options=["A", "B", "C"], title="Title").build()
```

Comment:
```suggestion
        checklist = Checklist(id="checklist", options=["A", "B", "C"], title="Title").build()
```
For simple cases like this it's best to make the test as strict as possible so to try and not specify any `keys_to_strip`. This means you'll need to supply an `id` here.